src/world/Objects/Units/Unit.cpp

```diff
@@ -75,7 +75,7 @@ Unit::Unit() :
 #endif
 
     // Zyres: initialise here because multiversion differences
-    std::fill_n(PctPowerRegenModifier, TOTAL_PLAYER_POWER_TYPES, 1);
+    std::fill_n(PctPowerRegenModifier, TOTAL_PLAYER_POWER_TYPES, '1');
```

Comment:
You are initializing float values with char '1' which is 49 in integer and that would be bad... Try with 1.0f

Reply:
up🎈

Reply:
Without quotes. Like this (PctPowerRegenModifier, TOTAL_PLAYER_POWER_TYPES, 1.0f);